scripts/setup.sh

```diff
@@ -0,0 +1,2 @@
+npm i
+./scripts/db-start.sh
```

Comment:
WARNING: Starting up the MySQL through Docker won't work if you have MySQL server installed since the ports conflict.